contracts/contracts/ccip/router/messages.tolk

```diff
@@ -111,6 +113,10 @@ struct (0x8ae25114) Router_MessageRejected {
     error: uint256;
 }
 
+struct (0xaf7a9ac6) Router_RMNOwnableMessage {
+    content: RemainingBitsAndRefs; // Ownable2Step_Messages
```

Comment:
Hmm this essentially double wraps the message right? Two opcodes. I guess it's the most straightforward